editor.planx.uk/src/pages/Preview/Node.tsx

```diff
@@ -96,13 +104,16 @@ const Node: React.FC<Props> = (props) => {
   });
 
   switch (props.node.type) {
+    case TYPES.AddressInput:
+      return <AddressInputComponent {...getComponentProps<AddressInput>()} />;
+
```

Comment:
This file is a bit extra noisy because I've alphabetized our component types within the `switch` block - only `Checklist`, `Filter`, `Question` types have functional changes!